main.cpp

```diff
@@ -0,0 +1,43 @@
+/**
+ * @file main.cpp
+ *
+ * @brief Program to ingest LLA position data and produce ECEF velocity points.
+ *
+ * @author Kendra Hale, <kendrahale98@gmail.com>
+ *
+ * @date 10/25/24
+ */
+#include <string>
+#include <vector>
+
+#include "./lla_to_ecef.h"
+
+int main() {
+  std::vector<PositionLLA> lla_data = read_csv_lla("./SciTec_code_problem_data.csv");
+
+  timespec point_of_interest1 {1532335268, 0};
+  timespec point_of_interest2 {1532334000, 0};
+
+  get_ecef_vel_at_poi(lla_data, point_of_interest1, true);
+  get_ecef_vel_at_poi(lla_data, point_of_interest2, true);
+
+// Exact match found for timestamp: 1532334879 s, 40000000 ns
```

Comment:
are these the results?

Reply:
if so why 3 results and only 2 calls?

Reply:
yeah need to remove these. had a result at a known point just for a sanity check before I wrote all the tests